### ISSUE TITLE
AttributeError for download_urls

### ISSUE THREAD

**Issue by kevinastone** [brettcannon/caniusepython3#95]
```
> caniusepython3 -r requirements.txt 
No handlers could be found for logger "ciu"
Finding and checking dependencies ...
[WARNING] Stale overrides: set([u'xlwt', u'jmespath'])
Traceback (most recent call last):
  File "$HOME/.local/bin/caniusepython3", line 11, in <module>
    sys.exit(main())
  File "$HOME/.local/venvs/caniusepython3/lib/python2.7/site-packages/caniusepython3/__main__.py", line 179, in main
    check(projects_from_cli(args))
  File "$HOME/.local/venvs/caniusepython3/lib/python2.7/site-packages/caniusepython3/__main__.py", line 167, in check
    blockers = dependencies.blocking_dependencies(projects, pypi.all_py3_projects())
  File "$HOME/.local/venvs/caniusepython3/lib/python2.7/site-packages/caniusepython3/dependencies.py", line 81, in blocking_dependencies
    dist = distlib.locators.locate(project)
  File "$HOME/.local/venvs/caniusepython3/lib/python2.7/site-packages/distlib/locators.py", line 361, in locate
    result.download_urls = versions.get('urls', {}).get(version, set())
AttributeError: 'dict' object has no attribute 'download_urls'
```

**Comment by brettcannon** [comment]
Are you using the latest release of distlib? And can you provide your requirements.txt file or figure out which project is triggering the error if you are not running the latest release of distlib?

**Comment by kevinastone** [comment]
Afraid I can't share the requirements.txt.  I'll see if I can isolate the problem line.

Here's my venv:
```
> pip freeze
argparse==1.3.0
caniusepython3==3.1.0
distlib==0.2.0
futures==3.0.2
wsgiref==0.1.2
```

**Comment by brettcannon** [comment]
I actually just remembered this was fixed as part of #87 , so it will get resolved in the next release of caniusepython3 (hopefully Friday).
